src/data/datatypes.js

```diff
@@ -857,9 +857,8 @@ const postgresTypesBase = {
       return field.default.length <= field.size;
     },
     hasCheck: true,
-    isSized: true,
+    isSized: false,
     hasPrecision: false,
-    defaultSize: 65535,
```

Comment:
how these changes would impact other database engines?

Reply:
This shouldn't impact any other database, because each database have their own {database}TypesBase.